src/contrib/cluster/Akka.DistributedData.Tests/LightningDb/LmdbDurableStoreSpec.cs

```diff
@@ -45,16 +46,54 @@ public void Lmdb_should_not_throw_when_opening_existing_directory()
                 var di = new DirectoryInfo(DDataDir);
                 di.Delete(true);
             }
+
             Directory.CreateDirectory(DDataDir);
+            var lmdb = ActorOf(LmdbDurableStore.Props(LmdbDefaultConfig));
+            lmdb.Tell(LoadAll.Instance);
+            ExpectMsg<LoadAllCompleted>();
+        }
 
-            var testKit = new TestKit.Xunit2.TestKit(BaseConfig, nameof(LmdbDurableStoreSpec), _output);
-            var probe = testKit.CreateTestProbe();
+        [Fact]
+        public void Lmdb_creates_directory_when_handling_first_message()
+        {
+            if (Directory.Exists(DDataDir))
+            {
+                var di = new DirectoryInfo(DDataDir);
+                di.Delete(true);
+            }
+
+            Directory.Exists(DDataDir).Should().BeFalse();
+            IActorRef lmdb = ActorOf(LmdbDurableStore.Props(LmdbDefaultConfig));
+            lmdb.Tell(LoadAll.Instance);          
+            AwaitCondition(() => HasMessages);
+            ExpectMsg<LoadAllCompleted>();
+            Directory.Exists(DDataDir).Should().BeTrue();
+        }
 
-            var config = testKit.Sys.Settings.Config.GetConfig("akka.cluster.distributed-data.durable");
-            var lmdb = testKit.Sys.ActorOf(LmdbDurableStore.Props(config));
-            lmdb.Tell(LoadAll.Instance, probe.Ref);
+        [Fact]
+        public void Lmdb_logs_meaningful_error_for_invalid_dir_path()
+        {
+            // Try to create a directory with a path exceeding allowed length 
+            string invalidName = new string('A', 247);           
```

Comment:
This test fails on Azure Pipeline because cloud's allowed directory path length exceeds 248 characters, and a folder is created, which shouldn't happen. Probably, it is better to remove this unit test completely because it relies on environment.